### PR TITLE
✅ Add visual diff tests for dead embeds

### PR DESCRIPTION
**Pull request by Enriqe**
Read #21402 for more context

#19213 #16522

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story-embedded-component.js [hunk 1]
```diff
@@ -115,15 +115,16 @@ function getComponentSelectors(components, opt_predicate) {
 }
 
 /** @const {string} */
-const INTERACTIVE_ATTR_NAME = '[interactive]:not([interactive=false])';
+const INTERACTIVE_EMBED_SELECTOR = '[interactive=""]';
```
Comment (Enriqe) [inline extensions/amp-story/1.0/amp-story-embedded-component.js]: I updated the selector here according to 

https://github.com/ampproject/amphtml/pull/21463#pullrequestreview-215877428

Comment (gmajoulet) [reply extensions/amp-story/1.0/amp-story-embedded-component.js]: Oh cool. Can you change it to simply `[interactive]` though?

**Comment by gmajoulet** [comment]
I can't find the new screenshot in Percy, could you please check if everything is working?
Also, what do you think of changing the "dead" word into "non interactive"? It'd be less creepy and more clear haha

File: extensions/amp-story/1.0/amp-story-embedded-component.js (hunk 1 above)
Comment (Enriqe) [reply extensions/amp-story/1.0/amp-story-embedded-component.js]: That would select attributes with values: https://jsfiddle.net/ryqokLn2/1/

Even if validation will not allow it I think we should be consistent and leave it like this.

Comment (gmajoulet) [reply extensions/amp-story/1.0/amp-story-embedded-component.js]: We don't need to check for things that aren't allowed by the validator though, but I don't feel strongly, so up to you : )

**Pull request merged**